packages/web/src/common/store/account/sagas.js

```diff
@@ -97,21 +100,32 @@ function* onSignedIn({ payload: { account } }) {
   const sentry = yield getContext('sentry')
   const analytics = yield getContext('analytics')
   if (account && account.handle) {
-    let solanaWallet = ''
-    try {
-      solanaWallet = (yield call(
-        getRootSolanaAccount,
-        audiusBackendInstance
-      )).publicKey.toBase58()
-    } catch (e) {
-      console.error('Failed to fetch Solana root wallet during identify()', e)
-    }
-    // Set analytics user context
+    const libs = yield call(audiusBackendInstance.getAudiusLibs)
+    const web3User = yield call([libs.Account, libs.Account.getWeb3User])
+
     const traits = {
       isVerified: account.is_verified,
-      trackCount: account.track_count,
-      solanaWallet
+      trackCount: account.track_count
     }
+
+    // If operating as a managed account, identify the manager user id
+    if (web3User && web3User.user_id !== account.user_id) {
+      traits.managerUserId = web3User.user_id
```

Comment:
Upside to doing it this way: Automatically attached to all events as a "user property"
Downside to doing it this way: Accounts with multiple managers will have this overwritten each time a new manager signs in such that the most recent one shows in the "properties" section for a given user.
I've confirmed that this doesn't affect the ability to split the events on the value of `managerUserId` or `managerHandle` though. So it's mostly a cosmetic issue in the user summary view in Amplitude.

Reply:
Yeah I think this will work fine, thanks for clarifying